.github/workflows/publish.yaml

```diff
@@ -0,0 +1,27 @@
+name: Publish
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - 'v*.*.*'
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    env:
+      CONTAINER_ENGINE: podman
```

Comment:
Please note that this project uses a different environment variable in the Makefile.

Reply:
you're right my bad :)